CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * [ObjectServer] Added `SyncSession.uploadAllLocalChanges()`.
 * Added `Nullable` annotation to methods that may return `null` in order to improve Kotlin usability. This also introduced a dependency to `com.google.code.findbugs:jsr305`.
 * Added support for new data type `MutableRealmIntegers`. The new type behaves almost exactly as a reference to a Long (mutable nullable, etc) but supports `increment` and `decrement` methods, which implement a Conflict Free Replicated Data Type, whose value will converge even when changed across distributed devices with poor connections. (#4266)
+* Added `SyncUser.allSessions()` to retrieve the all valid sessions belonging to the user. (#4783)
```

Comment:
@nhachicha @cmelchior Since ObjectStore's implementation returns sessions which are not in error, I think `valid` is more suitable for this PR.